ote/src/cljs/ote/ui/main_header.cljs

```diff
@@ -39,6 +39,22 @@
        (fn [_]
          [:span {:ref "sensor"}])})))
 
+(defn esc-press-listener [e! app]
+  "Listens to keydown events on document. If esc is clicked call CloseHeaderMenus"
+  (let [esc-press (fn [event]
+                      (if (= event.keyCode 27)
+                        (e! (fp-controller/->CloseHeaderMenus))))]
+    (r/create-class
+      {:component-did-mount
+       (fn [_]
```

Comment:
PR already reviewed and merged, but just a remark that these functions could have been aligned to single line.
Ref: https://github.com/bbatsov/clojure-style-guide#oneline-short-fn
No biggie :)